sdk/identity/tests.yml

```diff
@@ -1,26 +1,34 @@
 trigger: none
 
+resources:
+  containers:
+    - container: 'ubuntu_netcore_keyring'
+      image: 'azsdkengsys.azurecr.io/dotnet/ubuntu_netcore_keyring:523386'
+      endpoint: 'azsdkengsys'
+      options: -ti --cap-add=IPC_LOCK
+
 extends:
-  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     AdditionalPlatforms:
-      DisplayName: 'Test on Linux with keyring'
-      OSVmImage: "ubuntu-18.04"
-      TestTargetFramework: netcoreapp2.1
-      Container:
-        image: 'azsdkengsys.azurecr.io/dotnet/ubuntu_netcore_keyring:523386'
-        endpoint: 'azsdkengsys'
-        options: -ti --cap-add=IPC_LOCK
-      PreSteps:
-        - script: |
-            set -x
-            export $(dbus-launch)
-            gnome-keyring-daemon --start --daemonize --components=secrets
-            echo "##vso[task.setvariable variable=DBUS_SESSION_BUS_ADDRESS]$DBUS_SESSION_BUS_ADDRESS"
-            echo "##vso[task.setvariable variable=DBUS_SESSION_BUS_PID]$DBUS_SESSION_BUS_PID"
-            echo "##vso[task.setvariable variable=GNOME_KEYRING_CONTROL]$GNOME_KEYRING_CONTROL"
-    ServiceDirectory: identity    
+      Linux_NetCore_Keyring:
+        OSVmImage: "ubuntu-18.04"
+        TestTargetFramework: 'netcoreapp2.1'
+        Container: 'ubuntu_netcore_keyring'
+        SupportedClouds: 'Public,Canary,Preview'
+    ServiceDirectory: identity
     PreSteps:
+      - pwsh: Install-Module -Name Az -Scope CurrentUser -AllowClobber -Force -Verbose
```

Comment:
Is this new? Is it because the container doesn't have the Az command lines?

Reply:
I was rolling in a short-term workaround to get these working again for testing since they are broken. One of my next items was to just put the modules in the Dockerfile instead.

Reply:
(since we removed the install modules step in favor of the pre-installed modules on the agent filesystem, this got broken in the container context)